draft-packages/menu/KaizenDraft/Menu/StatelessMenu.tsx

```diff
@@ -113,6 +113,7 @@ export const StatelessMenu: React.FunctionComponent<StatelessMenuProps> = ({
   ) : null
 
   return (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
     <div data-automation-id={automationId} onClick={onClick}>
```

Comment:
Bit of a weird one, this `onClick` prop is only used twice (both within Kaizen) and it's only used to `stopPropagation`. I'm just going to ignore this, because I have plans to rewrite Menu :)